src/sentry_plugins/asana/plugin.py

```diff
@@ -202,7 +203,14 @@ def get_config(self, *args, **kwargs):
             client = self.get_client(user)
         except PluginIdentityRequired as e:
             self.raise_error(e)
-        workspaces = client.get_workspaces()
+        try:
+            workspaces = client.get_workspaces()
+        except Exception as e:
+            if (
+                e.response.reason == "Bad Request"
```

Comment:
@ceorourke Not every error is going to have a `response.reason` attribute (e.g. a ConnectionError)